traceloop-sdk/traceloop-sdk.gemspec

```
@@ -17,8 +17,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'opentelemetry-semantic_conventions_ai', '~> 0.0.3'
 
-  spec.add_dependency 'opentelemetry-sdk', '~> 1.3.1'
-  spec.add_dependency 'opentelemetry-exporter-otlp', '~> 0.26.1'
+  spec.add_dependency 'opentelemetry-sdk', '~> 1.10'
+  spec.add_dependency 'opentelemetry-exporter-otlp', '~> 0.31'
```

RuboCop (1.82.1)

[convention] Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `opentelemetry-exporter-otlp` should appear before `opentelemetry-sdk`.

(Gemspec/OrderedDependencies)